cpp/src/reductions/struct_minmax_util.cuh

```diff
@@ -21,7 +21,7 @@
 #include <cudf/detail/utilities/device_operators.cuh>
 #include <cudf/detail/utilities/vector_factories.hpp>
 #include <cudf/reduction/detail/reduction_operators.cuh>
-#include <cudf/table/row_operators.cuh>
+#include <cudf/table/experimental/row_operators.cuh>
```

Comment:
Also please remove the header at line 20.

Reply:
There are also a lot of unused headers such as at line 28-29. Please also remove them. See https://github.com/rapidsai/cudf/pull/10811/files#diff-7f4becb7510a70126c28187c4ecfdfe6498dcf8ee897be829b50b403f4868fbcL21-L29.